pyomo/contrib/mindtpy/iterate.py

```diff
@@ -3,7 +3,7 @@
 
 from pyomo.contrib.mindtpy.cut_generation import (add_oa_cuts, add_ecp_cuts)
 
-from pyomo.contrib.mindtpy.mip_solve import (solve_OA_master,
+from pyomo.contrib.mindtpy.mip_solve import (solve_MIP_master,
```

Comment:
we have `solve_MIP_master` and `handle_mip`. Let's settle for the notation. I'm open to suggestions, but maybe let's consider naming them just master and subproblem (and drop the MIP and NLP since it won't be true in general).